src/features/form/layout/formLayoutSlice.ts

```diff
@@ -33,6 +35,7 @@ export interface ILayoutState {
   error: Error | null;
   uiConfig: IUiConfig;
   layoutsets: ILayoutSets | null;
+  footerLayout: IFooterLayout | null;
```

Comment:
I think we could create an entirely new slice for the footer. In my head, the formLayout slice is tied to the process, while the footer is something we could (and should) fetch before we start thinking about the process/instance.

Reply:
Yup, I guess that could work! Thinking more about it, it might just be confusing to have the footer layout being loaded into the form layout state, when those two layout definitions have nothing to do with each other.